django/contrib/admin/options.py

```diff
@@ -962,7 +962,11 @@ def log_deletion(self, request, obj, object_repr):
             action_flag=DELETION,
         )
 
-    @display(description=mark_safe('<input type="checkbox" id="action-toggle">'))
+    @display(
+        description=mark_safe(
+            '<input type="checkbox" id="action-toggle" autocomplete="off">'
```

Comment:
I made use of https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocomplete